src/vue-components/inflections.vue

```diff
@@ -311,6 +317,17 @@
     },
 
     methods: {
+      updateWidth: function (timeout) {
```

Comment:
I'm wondering if timeout can be replaced with something else as I believe timeouts in situations like that can be unreliable. If we need a timeout to wait till a view is rendered by Vue, and it takes up to 400ms to render a simple view, it can take longer for more complex views. This will result in code to stop working on some views while working on others. It will also be a floating bug because it can render faster or slower depending on a machine and environment resulting in things being really hard to reproduce.

Are there any other solutions available? Can we use something like `Vue.nextTick()`?